src/game/scenes/arena.c

```diff
@@ -700,7 +700,7 @@ void arena_har_defeat_hook(int loser_player_id, scene *scene) {
         local->winner = winner_player_id;
         if(is_singleplayer(gs)) {
             player_winner->sp_wins |= 2 << player_loser->pilot->pilot_id;
-            if(player_loser->pilot->pilot_id == PILOT_KREISSACK) {
+            if(player_loser->pilot->pilot_id == PILOT_KREISSACK && player_loser->pilot->har_id == HAR_NOVA) {
```

Comment:
Technically this is behind a singleplayer check, so a hacked kreissack in tournament mode wouldn't trigger this code? I don't object to the additional check, but I don't know if it will actually fix anything a user is likely to experience.

Reply:
i have hacked around enough to be annoyed by it-- maybe i just like kreissack's "i'm not supposed to be on this side of the screen!" messages too much